src/Illuminate/Collections/MultipleItemsFoundException.php

```diff
@@ -6,4 +6,34 @@
 
 class MultipleItemsFoundException extends RuntimeException
 {
+    /**
+     * The number of items found.
+     *
+     * @var int
+     */
+    protected $count;
+
+    /**
+     * MultipleItemsFoundException constructor.
+     *
+     * @param  int  $count
+     * @param  int  $code
+     * @param  \Throwable|null  $previous
+     * @return void
+     */
+    public function __construct($count, $code = 0, $previous = null)
```

Comment:
This sounds like a breaking change? This exception now has a new type of required dependency which wasn't there before.

Reply:
Actually, it's not so bad I think. This should be good 👍